### PR TITLE
[18.0][MIG] mail_layout_preview: Migration to 18.0

### PR DESCRIPTION
**Pull request by Kimkhoi3010**

### CODE REVIEW

File: mail_layout_preview/readme/CREDITS.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The development and migration of this module from 15.0 to 16.0 has been
+financially supported by:
+
+- Camptocamp
+
+
```
Comment (chaule97) [inline mail_layout_preview/readme/CREDITS.md]:
```suggestion
```
remove it

Comment (nilshamerlinck) [reply mail_layout_preview/readme/CREDITS.md]: why remove traces from the past? 

> The development and migration of this module from 15.0 to 16.0 and from 17.0 to 18.0 has been
financially supported by:

**Comment by nilshamerlinck** [comment]
if `https://github.com/OCA/social/pull/1371` works, then you can take the occasion to approve it as well

**Comment by nilshamerlinck** [comment]
@chaule97 is this module still useful?

File: mail_layout_preview/readme/CREDITS.md (hunk 1 above)
Comment (chaule97) [reply mail_layout_preview/readme/CREDITS.md]: I thought when he append message `17.0 to 18.0`, we don't need old message above

**Comment by chaule97** [comment]
> @chaule97 is this module still useful?

yes, sorry I forgot about it

**Comment by nilshamerlinck** [comment]
why is it still useful?

**Comment by chaule97** [comment]
> why is it still useful?

I think someone might want to preview the template in full screen.
image

**Review by chaule97 — APPROVED** [review]

**Comment by Kimkhoi3010** [comment]
I created a new PR [OCA#1472](https://github.com/OCA/social/pull/1472) to OCA/social.
(This PR was closed auto after rebased)

**Pull request closed without merging**